Frends.RabbitMQ.Publish/README.md

```diff
@@ -0,0 +1,25 @@
+# Frends.RabbitMQ.Publish
+Frends RabbitMQ task to publish a string or byte array message to RabbitMQ queue.
+
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](https://opensource.org/licenses/MIT) 
+[![Build](https://github.com/FrendsPlatform/Frends.RabbitMQ/actions/workflows/Publish_build_and_test_on_main.yml/badge.svg)](https://github.com/FrendsPlatform/Frends.RabbitMQ/actions)
+![MyGet](https://img.shields.io/myget/frends-tasks/v/Frends.RabbitMQ.Publish)
+![Coverage](https://app-github-custom-badges.azurewebsites.net/Badge?key=FrendsPlatform/Frends.RabbitMQ/Frends.RabbitMQ.Publish|main)
+
+# Installing
+
+You can install the Task via frends UI Task View or you can find the NuGet package from the following NuGet feed https://www.myget.org/F/frends-tasks/api/v2.
+
+## Building
+
+Rebuild the project
+
+`dotnet build`
+
+Run tests
```

Comment:
Please add information here about spinning up docker container before running tests. You can also add the docker command here.